### PR TITLE
exit command/ generate points from database/mid points calculations

### PR DESCRIPTION
**Pull request by chiemeliecoder**
- Creation of an exit command to exit the program
- Creation of a method to generate five (5) points from the database
- Creation of a method that prints out the midpoints between the start and end points

### CODE REVIEW

**Review by AndrewTchaikovsky — APPROVED** [review]
We may want to consider creating the logic where the user picks start and end points (by id or name) instead of entering x and y coordinates of each point.

File: pom.xml [hunk 1]
```diff
@@ -64,15 +64,19 @@
...
     </dependency>
-      <dependency>
-        <groupId>org.apache.logging.log4j</groupId>
-        <artifactId>log4j-api</artifactId>
-        <version>2.20.0</version>
-      </dependency>
+
+    <dependency>
+      <groupId>log4j</groupId>
+      <artifactId>log4j</artifactId>
```
Comment (remynewton) [inline pom.xml:70]: Why are we changing the log4j dependency here? I thought Andrey fixed that. Also, if it continues to be a problem we can use a multiplatform docker image for it

File: src/main/java/org/example/App.java [hunk 2]
```diff
@@ -53,12 +94,104 @@ public static void main( String[] args ) {
...
+            if (startXInput.equalsIgnoreCase("exit")) {
+                exit = true;
+                continue;
+            }
+
+            double startXC = Double.parseDouble(startXInput);
+
+            System.out.print("Enter the y-coordinate of the start point: ");
+            double startYC = scan.nextDouble();
+            System.out.print("Enter the x-coordinate of the end point: ");
```
Comment (remynewton) [inline src/main/java/org/example/App.java:150]: I agree with Andrey. We need to enter points here not coordinates

**Review by remynewton — APPROVED** [review]

**Review by TomashevskiiVladimirSolvd — APPROVED** [review]
Great job.

**Pull request merged**